caluma/form/filters.py

```diff
@@ -11,6 +12,7 @@
 class FormFilterSet(FilterSet):
     search = SearchFilter(fields=("slug", "name", "description"))
     order_by = OrderingFilter(label="FormOrdering", fields=("name",))
+    meta__has_key = CharFilter(lookup_expr="has_key", field_name="meta")
```

Comment:
I would get rid of the double underscore here, as it leaks out to the graphQL representation, and is derived from an implementation (django queryset filter syntax).

Reply:
Done.

Reply:
I agree, `meta_HasKey` looks quite funny on the API. This feature will always be restricted to `meta` fields, so some an alternative name might be `metaHasKey`.

I'm wondering though: Can we imagine a way to filter for things like "meta property X equals/is bigger than/is smaller than Y"? Potentially, for multiple properties at once? Maybe something like

`filterMeta: '{"X": { "operator": "equals", "value": "Y" }, "Z": {...}}'